internal/provider/kubernetes/gateway.go

```diff
@@ -30,19 +30,19 @@ type gatewayReconciler struct {
 	log             logr.Logger
 
 	initializeOnce sync.Once
-	resourceTable  *ResourceTable
+	resources      *message.ProviderResources
```

Comment:
Note that https://github.com/envoyproxy/gateway/pull/196 adds `ResourceTable` coverage for kube provider integration tests. These tests will need to be updated for CI to pass.